addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -34,6 +34,69 @@ class ImLivechatChannelMemberHistory(models.Model):
     )
     avatar_128 = fields.Binary(compute="_compute_avatar_128")
 
+    # REPORTING FIELDS
+
+    session_country_id = fields.Many2one("res.country", related="channel_id.country_id")
+    session_livechat_channel_id = fields.Many2one(
+        "im_livechat.channel", "Live chat channel", related="channel_id.livechat_channel_id"
+    )
+    session_outcome = fields.Selection(
+        [
+            ("no_answer", "Never Answered"),
+            ("no_agent", "No one Available"),
+            ("no_failure", "Success"),
+            ("escalated", "Escalated"),
+        ],
+        compute="_compute_session_outcome",
+        store=True,
+    )
+    session_start_hour = fields.Float(
+        "Session Start Hour", compute="_compute_session_start_hour", store=True
+    )
```

Comment:
I guess the same applies to all "channel" values that are computed here:

already create the compute in channel in this PR instead of in history, and make it a related